components/at_mention/at_mention.jsx

```diff
@@ -48,6 +48,7 @@ export default class AtMention extends React.PureComponent {
     }
 
     handleClick = (e) => {
+        e.stopPropagation();
```

Comment:
Are there any side effect to how at_mention function when this feature is enabled?  Do the profile pop-over still works as expected?

Reply:
In my testing, I haven't seen any adverse side effects. QA will have to vet more extensively, but this was necessary to prevent the `at_mention` from bubbling up above the post click action.